### PR TITLE
Possible fix for first SMB-bug

### PR DESCRIPTION
**Pull request by szaimen**
Fix https://github.com/nextcloud/vm/issues/1074?

### CODE REVIEW

**Comment by szaimen** [comment]
@enoch85 this should be ready. Just if you wonder: I fixed the following travis issue with my solution:
```
In apps/smbmount.sh line 24:
if [ -z "$(grep "^hosts:" /etc/nsswitch.conf | grep wins)" ]
     ^-- SC2143: Use ! grep -q instead of comparing output with [ -z .. ].
```

**Comment by enoch85** [comment]
It's your script. :) Merge. :)

File: apps/smbmount.sh [hunk 1]
```diff
@@ -19,6 +19,13 @@ root_check
 # Install cifs-utils
 install_if_not cifs-utils
 
+# Make sure, that name resolution works
+install_if_not winbind
+if [ "$(grep "^hosts:" /etc/nsswitch.conf | grep wins)" == "" ]
```
Comment (enoch85) [inline apps/smbmount.sh]: Isn't it better to use `if [ -z .... ]` here?

Comment (enoch85) [reply apps/smbmount.sh]: So that would be something like `if [ -z "$(grep "^hosts:" /etc/nsswitch.conf | grep wins)" ]`

**Pull request merged**